planet/clients/subscriptions.py

```diff
@@ -97,18 +99,27 @@ async def list_subscriptions(self,
         Raises:
             APIError: on an API server error.
             ClientError: on a client error.
-
         """
+
+        class _SubscriptionsPager(_Pager):
+            """Navigates pages of messages about subscriptions."""
+            ITEMS_KEY = 'subscriptions'
+
+        params = {'status': [val for val in status or {}]}
+        url = URL('https://api.planet.com/subscriptions/v1', params=params)
+        req = self.session._client.build_request('GET', url)
+
         try:
-            # TODO: replace with httpx request.
-            async for sub in _server_subscriptions_get(status=status,
-                                                       limit=limit):
+            async for sub in _SubscriptionsPager(self.session._client,
+                                                 req,
+                                                 limit=limit):
```

Comment:
@jreiberkyle I've made the subscriptions pager an implementation detail here. I can't think of a reason why Paged needs to be part of this library's public API. Other than the async iterator magic methods its methods are all private. We could make the class itself private and reduce our API surface area and documentation needs. What do you think?